src/azul/plugins/metadata/hca/transform.py

```diff
@@ -989,8 +1070,55 @@ def transform(self) -> Iterable[Contribution]:
                                     map(self._sequencing_process, visitor.sequencing_processes.values())
                                 ),
                                 projects=[self._project(project)])
+                # Supplementary file matrices provide stratification values that
+                # need to be reflected by inner entities in the contribution.
+                if isinstance(file, api.SupplementaryFile):
+                    if Submitter.category_for(file) is not None:
```

Comment:
This would break if we added a 3rd category for which we don't want this to happen.